test/performance/infra/sender/http_load_generator.go

```diff
@@ -219,7 +217,7 @@ func (h HttpLoadGenerator) SendGCEvent() {
 	event.SetType(common.GCEventType)
 	event.SetSource(h.eventSource)
 
-	_, _, _ = h.ceClient.Send(context.TODO(), event)
+	_ = h.ceClient.Send(context.TODO(), event)
```

Comment:
Maybe the context should be Background here

Reply:
I am not reworking the code here, I am just updating to sdk v2